README.md

```diff
@@ -101,7 +102,7 @@ The schedule of the purge job, there can always only be one.
         "include_operations": "create,delete,pull"
     },
     "schedule": {
-        "cron": "0 0 0 * * *",
+        "cron": "53 0 0 * * *",
```

Comment:
Maybe we should set the second field (minutes) to a random/prim number on the examples instead of the seconds?

Reply:
Yes, this was a mistake.